### PR TITLE
Docker image for integration testing

### PR DESCRIPTION
**Pull request by ivankelly**
A docker image built with the pulsar distribution tarball. The same
image can be used to run zookeeper, bookkeeper, the broker or a
proxy. Scripts are provided to run each. There is also a script to
initialize the cluster, and another script to coordinate the
containers so that, for example, the broker doesn't start before the
cluster is initialized.

All services are run with supervisor, so that tests will be able to
start and stop the services.

### CODE REVIEW

File: tests/docker-images/latest-version-image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+FROM openjdk:8-jdk
+
+# Install some utilities
+RUN apt-get update && apt-get install -y netcat dnsutils supervisor python-kazoo
```
Comment (sijie) [inline tests/docker-images/latest-version-image/Dockerfile]: one high-level question here:

unlike bookkeeper, pulsar builds latest docker image from the package. is it possible to let latest docker image depends on the latest package? 

I mean depending on the image here: https://github.com/apache/incubator-pulsar/tree/master/docker

If so, that means we are testing the image as well, right?

Comment (merlimat) [reply tests/docker-images/latest-version-image/Dockerfile]: Yes, the image can be built on the same build (it's different from the BK docker image). 

I agree with Sijie that we should try to reuse same image, either by deriving from that or, even, to just put the additional scripts in the final image if needed.

Comment (ivankelly) [reply tests/docker-images/latest-version-image/Dockerfile]: If you're ok with that image being built via maven instead of scripts, that works for me.

**Comment by ivankelly** [comment]
@sijie @merlimat updated to use the standard image as the base image.

**Comment by ivankelly** [comment]
retest this please // flake

**Pull request merged**